### DISCUSSION TITLE
Error in execution of project

### DISCUSSION THREAD

**Discussion by madhurx** [honojs/hono#2473] [Q&A]
After running command
```
npm create hono@latest my-app

``` 
I chose **cloudflare-workers** template. Please check below image.
![image](https://github.com/honojs/hono/assets/116882181/a2f9ef62-3972-4825-8ade-c23875808dd7)
Then after when I run a command in my-app
```
npm run dev 
``` 
I got error as below
![image](https://github.com/honojs/hono/assets/116882181/df63b088-68dc-40fd-aee4-a7fa0dfdf991)
Please help to resolve the issue.
In other template like cloudflare-pages, it is working fine.

**Comment by NicoPlyley** [comment]
I believe this is a cloudflare workers issue I see someone recently also has this issue

From [stack overflow](https://stackoverflow.com/questions/78253344/npm-run-dev-not-working-in-wrangler-cloudflare-worker)

And last year ago from [GitHub](https://github.com/cloudflare/workers-sdk/issues/3698). Although that was fixed by updating I don't think this is related to Hono. Maybe try deleting the package and clearing cache, or try the older version of wrangler

**Reply by madhurx** [reply]
ok thank you. I have tried all by deleting the package, clearing cache and trying the older version of wrangler. Nothing worked.
I will try asking cloudflare.

**Reply by akshithere** [reply]
i am facing issue in the cloning template step itself but even if i choose aws-lambda instead of cloudfare-workers the error persists:
'''
⠋ Cloning the templateC:\Users\Akshit\AppData\Local\npm-cache\_npx\6bbb1d6f54609fb5\node_modules\create-hono\bin:17473
          throw new DegitError(`could not find commit hash for ${repo.ref}`, {
                ^

DegitError: could not find commit hash for main
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Degit.clone (C:\Users\Akshit\AppData\Local\npm-cache\_npx\6bbb1d6f54609fb5\node_modules\create-hono\bin:17337:11) {
  code: 'MISSING_REF',
  ref: 'main'
}
'''

**Comment by yasuaki640** [comment]
As of today, the issue appears to have been resolved.

```shell
npm create hono@latest test-3

> npx
> create-hono test-3

create-hono version 0.7.1
✔ Using target directory … test-3
? Which template do you want to use? cloudflare-workers
✔ Cloning the template
? Do you want to install project dependencies? yes
? Which package manager do you want to use? npm
✔ Installing project dependencies
🎉 Copied project files
Get started with: cd test-3
```

```shell
cd test-3/
```

```shell
npm run dev

> dev
> wrangler dev src/index.ts

 ⛅️ wrangler 3.55.0
-------------------
⎔ Starting local server...
[wrangler:inf] Ready on http://localhost:8787
╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [b] open a browser, [d] open Devtools, [l] turn off local mode, [c] clear console, [x] to exit                         │
[wrangler:inf] GET / 200 OK (5ms)
╭────────────────────────────────────────────────────────╮
│ [ open a   [d open    [l turn off   [c clear   [x to   │
│ ]browser,    Devtools,  local mode,   console,   exit  │
│                                                        │
╰────────────────────────────────────────────────────────╯
```
